<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewOutline</class>
 <widget class="QFrame" name="AddNewOutline">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="1" column="0" colspan="2">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="l_capture_method">
        <property name="text">
         <string>Capture Method::</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cmb_capture_method"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="l_capture_source">
        <property name="text">
         <string>Capture Source:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cmb_capture_source"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="l_lifecycle_stage">
        <property name="text">
         <string>Lifecycle Stage:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cmb_lifecycle_stage"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lb_suburb">
        <property name="text">
         <string>Suburb Locality:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="cmb_suburb"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lb_ta">
        <property name="text">
         <string>Territorial Authority: </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cmb_ta"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lb_town">
        <property name="text">
         <string>Town City:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cmb_town"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_save">
        <property name="maximumSize">
         <size>
          <width>85</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_reset">
        <property name="maximumSize">
         <size>
          <width>85</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cancel">
        <property name="maximumSize">
         <size>
          <width>85</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>fcb_ta</zorder>
     <zorder>fcb_suburb</zorder>
     <zorder>l_capture_method</zorder>
     <zorder>l_lifecycle_stage</zorder>
     <zorder>lb_ta</zorder>
     <zorder>cmb_lifecycle_stage</zorder>
     <zorder>lb_town</zorder>
     <zorder>fcb_town</zorder>
     <zorder>l_capture_source</zorder>
     <zorder>cmb_capture_method</zorder>
     <zorder>lb_suburb</zorder>
     <zorder>cmb_capture_source</zorder>
     <zorder>btn_reset</zorder>
     <zorder>btn_save</zorder>
     <zorder>btn_cancel</zorder>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
